jbpm-wb-integration/jbpm-wb-integration-backend/src/test/java/org/jbpm/workbench/wi/backend/server/casemgmt/service/CaseProvisioningIT.java

```diff
@@ -57,8 +57,7 @@ public static WebArchive create() {
                         "org.guvnor:guvnor-ala-registry-local",
                         "org.guvnor:guvnor-ala-services-api",
                         "org.guvnor:guvnor-ala-build-maven",
-                        "org.uberfire:uberfire-io",
-                        "org.sonatype.sisu.inject:guice-servlet"
```

Comment:
@psiroky Seems like it is working fine as it gets resolved as a transitive dependency.